### PR TITLE
Breakline at the end of the SARIF file.

### PR DESCRIPTION
**Pull request by BraisGabin**
Add a break-line at the end of the file.

I also did a little refactor to make the tests a bit easier.

### CODE REVIEW

File: src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt [hunk 1]
```diff
@@ -73,22 +73,22 @@ class SarifSerializerTest {
...
-            File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
-        )
+        val actual = SarifSerializer.fromJson(getResource("/test.sarif.json").readText())
         assertEquals(sarifSchema210, actual)
     }
-}
\ No newline at end of file
+}
+
+private fun getResource(path: String) = File(SarifSerializerTest::class.java.getResource(path)!!.path)
```
Comment (TWiStErRob) [inline src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt:94]: Don't assume File paths, there's better API without assumptions:
```suggestion
private fun getResource(path: String) = SarifSerializerTest::class.java.getResourceAsStream(path)!!.path

private fun InputStream.readText(): String =
    use { it.reader().readText() }
```

File: src/commonMain/kotlin/io/github/detekt/sarif4k/SarifSerializer.kt [hunk 2]
```diff
@@ -12,7 +12,7 @@ object SarifSerializer {
 
     fun toMinifiedJson(sarif: SarifSchema210): String = Json.encodeToString(sarif)
 
-    fun toJson(sarif: SarifSchema210): String = json.encodeToString(sarif)
+    fun toJson(sarif: SarifSchema210): String = json.encodeToString(sarif) + "\n"
```
Comment (TWiStErRob) [inline src/commonMain/kotlin/io/github/detekt/sarif4k/SarifSerializer.kt:15]: Does encodeToString use strictly `\n` or Platform-sensitive new lines? Match behaviour.

Comment (BraisGabin) [reply src/commonMain/kotlin/io/github/detekt/sarif4k/SarifSerializer.kt:15]: Looking at the tests, yes, they use `\n`. https://github.com/Kotlin/kotlinx.serialization/pull/2506/files#diff-e3bdf13d8a8ae78467a3aeecb08a9a8a4539a73ad45145a8ed9d1e43548ff6c8R64

File: src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt (hunk 1 above)
Comment (BraisGabin) [reply src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt]: Fixed

File: src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt [hunk 3]
```diff
@@ -76,22 +76,22 @@ class SarifSerializerTest {
...
-            File(SarifSerializerTest::class.java.getResource("/test.sarif.json").path).readText()
-        )
+        val actual = SarifSerializer.fromJson(getResource("/test.sarif.json").readText())
         assertEquals(sarifSchema210, actual)
     }
-}
\ No newline at end of file
+}
+
+private fun getResource(path: String): Reader = SarifSerializerTest::class.java.getResourceAsStream(path)!!.reader()
```
Comment (TWiStErRob) [inline src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt:97]: One very long line, wrap?
```suggestion
private fun getResource(path: String): Reader =
     SarifSerializerTest::class.java.getResourceAsStream(path)!!.reader()
```
(Please double check before commit, I wrote it on my phone and code editor was funny)

Comment (BraisGabin) [reply src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifSerializerTest.kt]: I did the change using Idea to avoid issues :P

**Pull request merged**